.github/workflows/ecosystem-ci-from-pr.yml

```diff
@@ -276,7 +276,7 @@ jobs:
             |-------|--------|----------------|
             ${rows.join("\n")}
 
-            Successful suites without change: ${successfulSuitesWithoutChanges.join(", ")}
+            ${conclusionEmoji.success} ${successfulSuitesWithoutChanges.join(", ")}
```

Comment:
I think the test wasn't needed at the end, it is clear enough to have the emoji here